src/api/java/mezz/jei/api/gui/handlers/IGuiContainerHandler.java

```diff
@@ -45,8 +45,23 @@ default Object getIngredientUnderMouse(T containerScreen, double mouseX, double
 	/**
 	 * Return the JEI-controlled clickable areas for this GUI.
 	 * This is useful when you want to add a spot on your GUI that opens JEI and shows recipes.
+	 *
+	 * @deprecated Prefer and implement {@link #getGuiClickableArea(ContainerScreen, double, double)} instead.
```

Comment:
Similarly, anything deprecated should start with `@deprecated since JEI version 1.2.3`